### PR TITLE
Feature/glue together published pipeline repo and existing script framework

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: script/src/main/java/org/opentestsystem/rdw/script/ScriptCompilers.java [hunk 1]
```diff
@@ -16,10 +21,26 @@
...
         return new ScriptCompiler(createGroovyClassLoader(sourceCode));
     }
 
+    /**
+     * Creates a script compiler for the given published pipeline
+     *
+     * @param publishedPipeline The published pipeline
+     * @return The script compiler for the pipeline
+     */
+    public static ScriptCompiler toScriptCompiler(final PublishedPipeline publishedPipeline) {
```
Comment (esotrope) [inline script/src/main/java/org/opentestsystem/rdw/script/ScriptCompilers.java]: extracted reused code - also i was going to have the reporting admin service use this to compile

File: script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java [hunk 2]
```diff
@@ -46,28 +45,27 @@ public static ScriptError convert(final ScriptRuntimeException e) {
      * @param e the CompilationFailedException
      * @return List of ScriptErrors, populated with the compile error messages, start lines, and start columns
      */
-    public static List<ScriptError> convert(final CompilationFailedException e) {
+    public static List<ScriptError> toScriptError(final CompilationFailedException e) {
```
Comment (malaffoon) [inline script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java]: `toScriptErrors`?

File: script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java [hunk 3]
```diff
@@ -46,28 +45,27 @@ public static ScriptError convert(final ScriptRuntimeException e) {
...
-            return convert((MultipleCompilationErrorsException)e);
+            return toScriptError((MultipleCompilationErrorsException)e);
         }
-
-        return Collections.singletonList(convert((GroovyRuntimeException)e));
+        return Collections.singletonList(toScriptError((GroovyRuntimeException)e));
     }
 
-    private static List<ScriptError> convert(final MultipleCompilationErrorsException e) {
+    private static List<ScriptError> toScriptError(final MultipleCompilationErrorsException e) {
```
Comment (malaffoon) [inline script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java]: ditto (plural)

File: script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java (hunk 2 above)
Comment (esotrope) [reply script/src/main/java/org/opentestsystem/rdw/script/ScriptErrors.java]: 👍

**Pull request merged**